util/dvsim/dvsim.py

```diff
@@ -266,6 +266,12 @@ def parse_reseed_multiplier(as_str: str) -> float:
     return ret
 
 
+def split_commas(as_str):
+    '''Split a string on commas, discarding empty strings'''
+    words = [w.strip() for w in as_str.split(',')]
```

Comment:
What happens if I pass something like `--run-opts "+myplusarg=a,b,c","+my_other_plusarg=val"`? UVM code has cases where a single plusarg value is in itself, a list of comma-separated values.

Reply:
That would definitely be a problem if we wanted to pass something with commas as a single "item", but I don't think we want to do that! In particular, we're using `split_commas` for the `--items` argument, which expects one or more test names.